### ISSUE TITLE
History functionality works, but History display appears inaccurate

### ISSUE THREAD

**Issue by vnnegt** [drawrowfly/tiktok-scraper#153]
**Describe the bug**

I use the `--store` option w/ custom history path ( `--historypath` ).

tiktok-scraper accurately respects these options, meaning if i delete an entry from a json file in the path, and scrape the account associated w/ the json file with the same options, it re-downloads the video associated with the deleted json entry.

the problem is that using the `history` command w/ the `--historypath` option does not appear to display an accurate representation of everything in the history path.

the listing does not show everything in the path, and for the history entries it does list, the downloaded posts column does not accurately reflect the number of entries in the associated json history file.

## If fields below will be empty then issue will be Ignored and Closed

**To Reproduce**
 I'm not sure how to reproduce this other than to say:

1. Scrape multiple accounts (dozens) using:
`tiktok-scraper user <USER> --number 0 --download --hd --noWaterMark --store --filepath "<FILE PATH>" --historypath "<HISTORY PATH>"`

2. Use `tiktok-scraper history` to view the history entries and verify that both the number of entries matches the number of json files in the history path and that the downloaded posts column value matches the number of entries in the json file.

- Used OS:
Windows 10

- NodeJs Version (node -v)
v13.12.0

- Scraper Version (tiktok-scraper --version)
1.1.11

- CLI or Module
???

- Used method
`history`

- Used options
`--historypath`

- Do you run the scraper through Proxy, VPN, VPS?
No

- If scraping from a profile. Did you checked if profile is Public? Scraper won't work with Private profiles!

- Preferably provide URL to the profile,hashtag,video and etc page

**Screenshots**

**Comment by drawrowfly** [comment]
"Use tiktok-scraper history to view the history entries and verify that both the number of entries matches the number of json files in the history path and that the downloaded posts column value matches the number of entries in the json file"

When you set custom history file path you also need to provide it with the **history** command 

I just executed your command set few times and after

tiktok-scraper history --historypath "<HISTORY PATH>"

Received accurate data

**Comment by vnnegt** [comment]
Right, I do use the `--historypath` option with the `history` command. If I didn't, the history command wouldn't output anything because I don't store history anywhere else.

I wish I knew what to do to reproduce this.

I think this is where things got weird:

1. I accidentally moved the folder i set my history path to.

2. I scraped with:
`tiktok-scraper user <USER> --number 0 --download --hd --noWaterMark --store --filepath "<FILE PATH>" --historypath "<HISTORY PATH>"`

but the folder the path was pointing to had been moved

3. I moved the folder back and scraped again with the same command + options ( i always use these same options via bash function )

4. This is when i noticed things were off

What's strange is that the `--store` / `--historypath` options work and respect everything in the path, but the actual history display is inaccurate when i use `tiktok-scraper history --historypath`

Is this display cached/indexed anywhere? Because it definitely does not show everythingin my history path.

The path has 217 json files and the history display only shows 78 entries
